src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/CaseServiceImpl.java

```diff
@@ -95,10 +95,12 @@
       "All Northern Ireland calls from CE Managers are to be escalated to the NI management team.";
   private static final String UNIT_LAUNCH_ERR_MSG =
       "A CE Manager form can only be launched against an establishment address not a UNIT.";
+  public static final String CCS_CASE_ERROR_MSG = "Operation not permissible for a CCS Case";
   private static final List<DeliveryChannel> ALL_DELIVERY_CHANNELS =
       List.of(DeliveryChannel.POST, DeliveryChannel.SMS);
 
   private static final String SCOTLAND_COUNTRY_CODE = "S";
+  public static final String CENSUS = "CENSUS";
```

Comment:
I think we should delete this CENSUS variable, and use **appConfig.getSurveyName()** instead.

Reply:
This constant removed and used appConfig.getSurveyName()